messaging/link-shortening/link-shortening-domain-cert/link-shortening-domain-cert.9.x.java

```diff
@@ -0,0 +1,73 @@
+// Install the Java helper library from twilio.com/docs/java/install
+
+import com.twilio.Twilio;
+import com.twilio.rest.messaging.v1.DomainCerts;
+
+public class Example {
+    // Find your Account SID and Auth Token at twilio.com/console
+    // and set the environment variables. See http://twil.io/secure
+    public static final String ACCOUNT_SID = System.getenv("TWILIO_ACCOUNT_SID");
+    public static final String AUTH_TOKEN = System.getenv("TWILIO_AUTH_TOKEN");
+
+    Sring certAndPrivateKey = """-----BEGIN CERTIFICATE-----
```

Comment:
Just curious, it has 3 quotation marks  (""") Is this expected ?

Reply:
Ah, this is the JDK15 "text blocks" feature. Is there a better, preferred way to deal with multiline strings in Java?

Reply:
Ah that makes sense!